app/services/govuk_bank_holiday_retriever.rb

```diff
@@ -10,7 +10,7 @@ def dates(group)
 private
 
   def response
-    store = ActiveSupport::Cache.lookup_store(:file_store, "/tmp/cache", expires_in: 10.days)
+    store = ActiveSupport::Cache.lookup_store(:file_store, "tmp/cache", expires_in: 10.days)
```

Comment:
@starswan This change is not a part of this ticket. We are already handling in other [PR#450](https://github.com/ministryofjustice/cfe-civil/pull/450)